admin_guide/quota.adoc

```diff
@@ -149,6 +149,10 @@ A *BestEffort* scope restricts a quota to limit the following resources:
 
 - `*pods*`
 
+*BestEffort* scoped pods can take advantage of slack compute resources in their
+namespace, and then require that users make an explicit resource request for
+pods that require a higher quality of service.
```

Comment:
or just link to https://docs.openshift.org/latest/admin_guide/overcommit.html#qos-classes ?